src/EdFi.AnalyticsMiddleTier.DataStandard32/RLS/MSSQL/0007-View-UserAuthorization-Create.sql

```diff
@@ -55,7 +55,11 @@ SELECT DISTINCT staffToScopeMap.StaffUniqueId AS UserKey
 		WHEN 'AuthorizationScope.District'
 			THEN 'ALL'
 		ELSE CAST(staffToScopeMap.EducationOrganizationId AS VARCHAR)
-		END AS SchoolPermission
+		END AS SchoolPermission,
+    CASE staffToScopeMap.UserScope
+        WHEN 'AuthorizationScope.District'
+            THEN staffToScopeMap.EducationOrganizationId
+        END AS DistrictId
```

Comment:
For the DataStandard32, add these lines in the version for postgresql (RLS / PostgreSQL / 0007-View-rls_UserAuthorization-Create.rls)

Reply:
Yes, we need to add these changes to the Postgres version.

Reply:
This is a good piece of feedback and initiated a quick internal discussion on how we should handle feature changes that affect multiple technology delivery options.  

Let's move this pull request into a feature branch and then the Ed-Fi team can make the modifications to include Postgres support.  We need the Postgres support added but don't want to add the burden to the contributing members who don't have that technology in their stack.

Reply:
I'm going to accept the pull request into the feature branch. Thank you @arturohernandez10.